pmlc/dialect/op_lib/BUILD

```diff
@@ -27,10 +27,55 @@ PLATFORM_COPTS = select({
     ],
 })
 
+plaidml_cc_library(
+    name = "OpModel",
+    srcs = [
+        "OpModel.cc",
+    ],
+    hdrs = [
+        "OpModel.h",
+    ],
+    deps = [
+        "@mlir//:TableGen",
+    ],
+)
+
+plaidml_cc_library(
+    name = "OpCPP",
+    srcs = glob([
+        "cpp/ImplGen.cc",
+        "cpp/WrapperGen.cc",
+    ]),
+    hdrs = glob([
+        "cpp/ImplGen.h",
+        "cpp/WrapperGen.h",
+        "cpp/utils.h",
+    ]),
+    deps = [
+        ":OpModel",
+        "@mlir//:TableGen",
+    ],
+)
+
+plaidml_cc_library(
+    name = "OpPY",
+    srcs = glob([
+        "python/WrapperGen.cc",
+    ]),
+    hdrs = glob([
+        "python/WrapperGen.h",
+        "python/utils.h",
+    ]),
+    deps = [
+        ":OpModel",
+        "@mlir//:TableGen",
+    ],
+)
```

Comment:
The 'bazel' way of doing this would be to put a `BUILD` file in `python` and `cpp` subdirs.